stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -351,6 +474,10 @@ private void validateKey(@NonNull @Size(min = 1) String publishableKey)
         }
     }
 
+    private void sourceTaskPostExecution(ResponseWrapper result, SourceCallback callback) {
```

Comment:
Oops -- no, I was mirroring what I did for cards and then realized I didn't need that method. Will remove.